packages/ui/src/ui/store/actions/settings/index.ts

```diff
@@ -131,10 +135,12 @@ export function reloadUserSettings(login: string): SettingsThunkAction {
     return async (dispatch, getState) => {
         try {
             const state = getState();
+            const cluster = getSettingsCluster(state);
+            const ytAuthCluster = state.global.ytAuthCluster || cluster;
```

Comment:
`state.global.ytAuthCluster` should be empty after success login, we don't need it here.

Reply:
Unfortunately, the reloadUserSettings function is called before we have information about the cluster.